certified-connectors/Tribal/README.md

```diff
@@ -0,0 +1,55 @@
+# Tribal Connector
+
+Tribal Edge modules, such as Tribal Admissions, provide staff and students the tools they need to succeed in the world of education. Connect to Tribal Edge modules using the Tribal Connector to manage information, create clearance checks for applications, and much more.
+
```

Comment:
@mszy-tribal Please refer the template provided for readme.md file and add required sections.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
these updates have been made

Reply:
@asitedeveloper The publisher section is missing.